core/platform/email/gae_email_services.py

```diff
@@ -63,13 +84,14 @@ def send_mail(
     if not feconf.CAN_SEND_EMAILS:
         raise Exception('This app cannot send emails.')
 
-    if not mail.is_email_valid(sender_email):
-        raise ValueError(
-            'Malformed sender email address: %s' % sender_email)
     if not mail.is_email_valid(recipient_email):
         raise ValueError(
             'Malformed recipient email address: %s' % recipient_email)
 
+    if not _is_sender_email_valid(sender_email):
```

Comment:
I think, in previous comments (which you haven't replied to?), @DubeySandeep was talking about updating mail.is_email_valid in general, rather than doing something specific for sender emails as opposed to recipient emails. (@DubeySandeep, is that correct?)

Reply:
I had raised this issue with the docstring provided in the gae_email_services.
![image](https://user-images.githubusercontent.com/16653571/55704944-2988b800-59fb-11e9-9445-d5ea7aaa8f5e.png)

In the above docstring, we had only asked for the sender email.
I'm not sure whether we should have such it for recipient email too? (As we only have the username (not the real name) of the users)

Reply:
Oh, I see. Yes, good point -- thanks! I am happy with either just using the recipient email, or the username followed by the recipient email. I'm not sure which is better.

Reply:
> EDIT: I'm not sure whether we can have user's Name from their Google authentication API.

No, let's not do that. See my previous response -- either is fine. Thanks for clarifying!

Reply:
Thanks @seanlip @DubeySandeep for clarifying this.